frontend/.env.local.sample

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_BACKEND_URI=http://ec2-3-64-179-99.eu-central-1.compute.amazonaws.com/
```

Comment:
Jirko, chceme mit v repozitari (resp. sample souboru) takoveto udaje?

Reply:
Je to nějaký security leak? 
Za mě je to asi ok, protože uri backendu je člověk schopný získat potom z každého requestu na webu. Ale pokud je tam něco co přehlížím a vznikala by tím nějaká hrozba tak určitě ne.

Reply:
Jsem paranoidnejsi, takze jakekoliv prozrazeni zdroje mi prijde potencionalne nebezpecne.

Je to podle me ale i zavadejici. Ted nam sandbox jede na konkretni EC2 instanci, ale Jakub naznacoval, ze se to asi pozdeji rozjede na ECS. Obecne povazuju AWS virtualky za jepice, ktere zitra muzou jet uplne jinde. Casem  budeme mit minimalne 2 prostredi (produkci/sandbox), pro kazdou bude jina domena/ip adresa.

Zkratka, do env souboru bych to nedaval. Radeji zadokumentovat do README ci INSTALL docky.

Reply:
To dává smysl. Dám to pryč, stejně je to pinnutá adresa na slacku.